index.html

```diff
@@ -1,16 +1,16 @@
 ---
 layout: layout.liquid
-title: Open source key-value database on blockchain
+title: Open Source on Blockchain Immutable Meta/Data
 hide_change: hidden
 ---
 
 <main role="main">
   <div class="jumbotron">
     <div class="layer"></div>
     <div class="container kva-container">
-      <h1 class="kva-header">Powerful blockchain data store, easy <span class="kva-em">and useful</span></h1>
+      <h1 class="kva-header">Blockchain Storage for Metadata and Text <span class="kva-em">Simple & Useful</span></h1>
       <p class="kva-intro">
-        A public key-value store and cryptocurrency, for developers and everyone else.
+        A public and permanent store for Metadata, Blogs, Text and Much More with Native Cryptocurrency.
```

Comment:
Capitalize on all words?

A Public and Permanent Store for Metadata, Blogs, Text and Much More with Native Cryptocurrency.